pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiKitRules.java

```diff
@@ -39,16 +37,16 @@
      * @return the base id of the kit rule inserted
      */
     default Long insertKitRuleByType(KitRuleType kitRuleType, Long pexOrCountryId) {
-        Long ruleTypeId = getRuleTypeIdFromCode(kitRuleType);
-        Long ruleId = insertRule(ruleTypeId);
-
+        long ruleId = insertRule(kitRuleType);
         switch (kitRuleType) {
             case COUNTRY:
                 insertCountryRule(ruleId, pexOrCountryId);
                 break;
             case PEX:
                 insertPexRule(ruleId, pexOrCountryId);
                 break;
+            case ZIP_CODE:
+                throw new DaoException("do not use for kit type " + kitRuleType);
```

Comment:
This should be okay. These are only used in some tests. I will refactor these DAOs later.